bring-it-up/bring-it-up
# BE#56 Add internal schools endpoint

## Description
#### Summary:
- Adds an internal endpoint `PUT /schools` to allow populating the database with schools entries from `schools.json`

#### Changes:
- Added `PUT /schools` endpoint
- Renamed `school.ts` to `school.service.ts`
- (unrelated) Fixed bug in `PUT /counselling-services` endpoint from previous linting changes

#### How to test this PR:
- In Postman, re-import the `bring-it-up.postman_collection.json` collection
- Run the `PUT /schools` endpoint (body should already be populated with the content inside `schools.json`)
- The schools should be added to the DB successfully
  
## Checklist
